azure-pipelines.yml

```diff
@@ -69,8 +91,10 @@ steps:
   displayName: '$(TEST_TARGET) Test'
   condition: eq( variables['Agent.OS'], 'Darwin')
 - script:  |
-    docker run --rm --user='root' -v `pwd`:/home/axom $(Compiler_ImageName) chown -R axom /home/axom
-    docker run --rm  -v `pwd`:/home/axom -e TEST_TARGET -e COMPILER -e DO_BUILD -e DO_TEST -e CMAKE_EXTRA_FLAGS $(Compiler_ImageName) ./scripts/azure-pipelines/linux-build_and_test.sh
+    echo " -e $TEST_TARGET -e $COMPILER -e $DO_BUILD -e $DO_TEST -e $CMAKE_EXTRA_FLAGS $(Compiler_ImageName) ./scripts/azure-pipelines/linux-build_and_test.sh"
+    docker run --rm --user='root' -v `pwd`:/home/axom/axom $(Compiler_ImageName) chown -R axom /home/axom
```

Comment:
You can actually run the docker container directly with the azure job spec, here is an example:

```
  - job: Ubuntu_Cuda
    pool:
      vmImage: 'ubuntu-16.04'
    container: alpinedav/ascent-ci:cuda-9.2-dev
```

Not sure if that really simplifies things or not in this case, but I wanted to mention it, since I saw we are calling docker ourselves to launch the container.